app/models/template.rb

```diff
@@ -1,7 +1,8 @@
 class Template < ActiveRecord::Base
   validates_lengths_from_database
 
-  attr_accessible :name, :template, :snippet, :audit_comment, :location_ids, :organization_ids, :locked, :default
```

Comment:
The attr can still be set at creation, but not mass-assigned, that's all.  The fix you put in should still work.

I don't think FG uses mass assignment, but haven't double checked.  I think functional test coverage is all we have to pick up these issues.

> Why prevent mass assignment? The auth check would still happen.

Indeed, which is why it could be added back.  The benefit of leaving it out would be giving us an extra layer of security in case the auth check was flawed.  (A user with just edit permission from specifying 'locked' in an API/UI update would be ignored automatically.)

A third way would be to use the `:as` option (and even `without_protection`) so we could do `:as => :dbseed` and permit that attribute to be assigned in that role.

I'd prefer to keep it unlisted or use a role/:as because of the security benefits of protected attrs (Foreman has had little/no protection until now), but if we must, then it could go back.